src/pages/layer-2.js

```diff
@@ -258,19 +258,18 @@ const Layer2Page = ({ data }) => {
   }, [intl.locale])
 
   const heroContent = {
-    title: "Layer 2",
-    header: "Ethereum for everyone",
-    subtitle:
-      "Scaling Ethereum without compromising on security or decentralization.",
+    title: translateMessageId("layer-2-hero-title", intl),
+    header: translateMessageId("layer-2-hero-header", intl),
+    subtitle: translateMessageId("layer-2-hero-subtitle", intl),
     image: getImage(data.heroImage),
-    alt: "test",
+    alt: translateMessageId("layer-2-hero-alt-text", intl),
     buttons: [
       {
-        content: "What is layer 2",
+        content: translateMessageId("layer-2-hero-button-1", intl),
         pathId: "what-is-layer-2",
       },
       {
-        content: "Use layer 2",
+        content: translateMessageId("layer-2-hero-button-2", intl),
         pathId: "use-layer-2",
         isSecondary: "isSecondary",
       },
```

Comment:
Oops missed this one 

<img width="324" alt="Screenshot 2022-05-30 at 15 39 54" src="https://user-images.githubusercontent.com/62268199/171015942-9784362f-f4be-48d0-834b-772c171c3fd8.png">